include/aws/auth/credentials.h

```diff
@@ -266,6 +266,32 @@ struct aws_credentials_provider_sts_options {
     aws_io_clock_fn *system_clock_fn;
 };
 
+/**
+ * The process credentials provider sources credentials from running a command or process.
+ * The command is configured in config file with the config key as "credential_process".
+ * On successfully running the command, the output should be a json data with the following
+ * format:
+ * {
+    "Version": 1,
+    "AccessKeyId": "accesskey",
+    "SecretAccessKey": "secretAccessKey"
+    "SessionToken": "....",
+    "Expiration": "2019-05-29T00:21:43Z"
+   }
+ * Version here identify the command output format version. If it involved in future, SDK should update
```

Comment:
"identifies" in first sentence.  Remove the rest of the paragraph and replace with "This provider is not a part of the default provider chain."  I know I do it sometimes, but I feel like "If X happens in the future then we should do Y" notes to ourselves should stay out of the public API documentation.